### DISCUSSION TITLE
Define attributes needed for data-driven groupings

### DISCUSSION THREAD

**Discussion by ASL-rmarshall** [cdisc-org/analysis-results-standard#142] [Ideas]
A data-driven grouping is a grouping where the values to be included in the analysis are not prespecified as part of the definition of the analysis.  Things to be clarified:
- If you include a specification of groups as part of the design of the analysis, do you want to be able to indicate whether:
  - All the defined groups are displayed (with zero values for groups that did not occur).
  - Only groups that appear in the data are displayed?
- Do you need to be able to specify a "display format" for data-driven group values (e.g. if this data value occurs, this is how it should be displayed)?  If so, is this any different to prespecified groups?
- How should dummy values for data-driven groups (needed for creation of the display shell) be defined?

**Comment by drewcdisc** [comment]
These questions need to be move to Discussions and then we'll add PBIs to the Backlog once we've answered the questions.

**Comment by drewcdisc** [comment]
@ASL-rmarshall @bhavinbusa I converted Issue #67 into Discussion #142.  After all the questions in here are answered we can create independent PBIs that may arise from the answers.
